mlflow/pyfunc/scoring_server/__init__.py

```diff
@@ -328,9 +329,13 @@ def get_cmd(
 ) -> Tuple[str, Dict[str, str]]:
     local_uri = path_to_local_file_uri(model_uri)
     timeout = timeout or MLFLOW_SCORING_SERVER_REQUEST_TIMEOUT.get()
+
+    if isinstance(host, str) and " " in host:
```

Comment:
Again, it does not help to check if there is a whitespace in the host parameter, we can just supply a payload without one, e.g. `localhost&id&blablabla`, so a use of shlex.quote is highly advised